doc/source/examples/ale_regression_california.ipynb

```diff
@@ -0,0 +1,875 @@
+{
```

Comment:
<p>There are some empty axes. Not a big deal, but would be great to remove them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>Yup, forgot to update.</p>